src/main/resources/avro/biodata.avdl

```diff
@@ -0,0 +1,133 @@
+@namespace("org.ga4gh.models")
+
+/**
+This protocol defines the "biodata" objects, which can be considered data
+representations of biological correlates.
+*/
+
+protocol Biodata {
```

Comment:
Still, that doesn't make Biodata a good name. May I suggest Specimen, which describes both individuals and samples?